main/EDTypesMod.F90

```diff
@@ -713,8 +233,8 @@ module EDTypesMod
   type, public :: ed_site_type
      
      ! POINTERS  
-     type (ed_patch_type), pointer :: oldest_patch => null()   ! pointer to oldest patch at the site  
-     type (ed_patch_type), pointer :: youngest_patch => null() ! pointer to yngest patch at the site
```

Comment:
Even though this is the FATES model, it is built on top of the Ecosystem Demography scaling approach, which at its core is patches and cohorts.  I think it is a good think to maintain the "ed_patch_type" and "ed_cohort_type" here, so that people are aware of its heritage.

Reply:
Okay, I thought in a FATES dev meeting we had discussed and agreed to change to `fates_` etc. I can change back though. @glemieux @ckoven @rosiealice

Reply:
after team discussion, I'm dropping this suggestion.  There are better/other ways to recognize the underpinnings of FATES, using the namespace here would be confusing